modules/new_profile/php/profile.class.inc

```diff
@@ -120,8 +135,126 @@ class Profile extends \NDB_Page
                 ->withStatus(405)
                 ->withHeader("Allow", "GET,POST");
         }
+    }
 
+    /**
+     * Processes the values and saves to database
+     *
+     * @param array $values the user data received..
+     *
+     * @return array
+     */
+    function processValidity($values)
+    {
+        // Validate (dateTaken, dateTakenConfirm, sex) not set.
+        if (!isset($values['dateTaken'])
```

Comment:
isset accepts multiple arguments:

`if (!isset($values['dateTaken'], $values['dateTakenConfirm'], $values['sex']))...`

Reply:
Hey @johnsaigle, how do you do this with not having phpcs complain about the line being too long? I've tried line breaks but then it complains about needing a boolean and other stuff. Could you try it and see if there is a way to satisfy phpcs.

Reply:
You can just leave it if PHPCS is having a big issue with it.

Reply:
you could also split the newlines on comma (unless PHPCS complains about that.)